package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "arweavekit",
-  "version": "1.4.9",
+  "version": "1.5.0",
```

Comment:
This can be bumped whenever we will release, not necessarily in this particular PR

Reply:
Nice